Romina.Api/Startup.cs

```diff
@@ -3,14 +3,21 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
+using Romina.Api.Repositories;
 
 namespace Romina.Api
 {
     public class Startup
     {
-        public Startup(IConfiguration configuration)
+        public Startup(IHostingEnvironment env)
         {
-            Configuration = configuration;
+            var builder = new ConfigurationBuilder()
```

Comment:
hey bud you dont need to do this here 

this is my fault but the newer version of net core does all this for you in the `Program.cs`.

so please revert this change in the constructor